### DISCUSSION TITLE
Collecting page data issue -11

### DISCUSSION THREAD

**Discussion by Zang983** [vercel/next.js#57788] [Help]
### Summary

I am writing to you because I have a problem during the build of my NextJS app.
I have hosting on the o2switch platform, which offers virtual servers with a Cpanel administration interface.

I want to put online a portfolio consisting of a single page and a single API call.

The backend is on the server, and the requests are working.
Locally, everything is also working, but since I don't have the same system (cloudlinux), I can't really compare.

I looked before writing this message, and I found that it's related to optimizations that I could disable. The project is lightweight, so turning off these optimizations shouldn't be problematic. However, I want to learn and understand this issue.

Edit : The version of Next.js is 13.5.4, and the version of Node.js is 18.16. I cannot upgrade to a newer version of Node.js.

Do you have any suggestions or leads to help me?

### Additional information

```js
Here is the result of my build that is having an issue at the "collecting page data" moment:
✓ Linting and checking the validity of types
uncaughtException Error: spawn /opt/alt/alt-nodejs18/root/usr/bin/node EAGAIN
at ChildProcess._handle.onexit (node:internal/child_process:283:19)
at onErrorNT (node:internal/child_process:476:16)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
errno: -11,
code: 'EAGAIN',
syscall: 'spawn /opt/alt/alt-nodejs18/root/usr/bin/node',
path: '/opt/alt/alt-nodejs18/root/usr/bin/node',
spawnargs: [
'/home/lejo4797/nodevenv/portfolioV1/18/lib/node_modules/next/dist/compiled/jest-worker/processChild.js'
]
```

**Comment by icyJoseph** [comment]
Could it be that you are running out of memory on that instance? How much memory available do you have in o2switch?

**Comment by Zang983** [comment]
> Could it be that you are running out of memory on that instance? How much memory available do you have in o2switch?

I have 48GB of RAM, no theoretical disk space limit. Even when trying to build the project without importing my various components, it fails. It's really strange.

**Comment by Zang983** [comment]
After several new searches, here is a solution that worked:
https://kdaws.com/learn/fixing-next-js-error-spawn-errno-11-on-cpanel-and-plesk-with-cloudlinux/

**Reply by ahmedtehseen** [reply]
thanks a bunch for saving my time brother.

**Reply by awyyyn** [reply]
This works for me as well using next 14.2.17, Thanks!
